tests/components/configurator/test_init.py

```diff
@@ -38,21 +37,27 @@ def test_request_least_info(self):
         state = states[0]
 
         assert configurator.STATE_CONFIGURE == state.state
-        assert request_id == state.attributes.get(configurator.ATTR_CONFIGURE_ID)
+        assert request_id == state.attributes.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)